components/collapsible-panel/collapsible-panel.js

```diff
@@ -315,6 +321,7 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 			'has-summary': this._hasSummary,
 			'has-before': this._hasBefore,
 			'scrolled': this._scrolled,
+			'group-last': this.parentElement.nodeName === 'D2L-COLLAPSIBLE-PANEL-GROUP' && Array.from(this.parentElement.children).indexOf(this) === this.parentElement.children.length - 1,
```

Comment:
I _think_ we can use `children` in this way...from the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/children#value)

> An [HTMLCollection](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection) which is a **live, ordered collection of the DOM elements** which are children of node.